lib/model.d.ts

```diff
@@ -2497,9 +2497,9 @@ export abstract class Model {
    * @param options.reset Clear all previously set data values
    */
   set<K extends keyof this>(key: K, value: this[K], options?: SetOptions): this;
-  set(keys: Partial<this>, options?: SetOptions): this;
+  set(keys: object, options?: SetOptions): this;
```

Comment:
`update` does, since it's calling `save`, which handles nested creations etc.

But this does not apply to `increment` and `decrement`. Gonna revert it 👍